extendeddatacrossword.go

```diff
@@ -140,7 +142,7 @@ func (eds *ExtendedDataSquare) solveCrosswordRow(
 	}
 
 	// Check that rebuilt shares matches appropriate root
-	err = eds.verifyAgainstRowRoots(rowRoots, uint(r), rebuiltShares)
+	err = eds.verifyAgainstRowRoots(rowRoots, uint(r), rebuiltShares, noShareInsertion, nil)
```

Comment:
Nit: I am not sure `verifyAgainstRowRoots` needs both `rebuiltIndex == -1` and `rebuiltShare == nil` to indicate computing the shares root without a previously rebuilt share.